src/python/WMCore/MicroService/Unified/MSOutput.py

```diff
@@ -20,11 +20,11 @@
 from WMCore.MicroService.Unified.MSCore import MSCore
 from WMCore.MicroService.Unified.Common import gigaBytes
 from WMCore.Services.CRIC.CRIC import CRIC
-from Utils.EmailAlert import EmailAlert
```

Comment:
Erik, can I suggest you to add a note to the top level docstring of:
https://github.com/dmwm/WMCore/blob/master/src/python/Utils/EmailAlert.py#L2

stating that it does not work from kubernetes pods? Maybe even add a reference to the GH issue reporting that issue.